README.md

```diff
@@ -1,9 +1,86 @@
 # dgc-dotnet
-C# /.NET implementation of the [Electronic Health Certificate Specification](https://github.com/ehn-digital-green-development/ehn-dgc-schema) to create and decode Digital Green Certificates
+C# /.NET implementation of the [Electronic Health Certificate Specification](https://github.com/ehn-digital-green-development/ehn-dgc-schema) to create and decode Digital Covid Certificates
 
 ## About
-This is a dotnet standard library. 
-It's meant to be a library to be used within other dotnet projects
+This is a dotnet standard library.
+
+## How to use
+### Verifing a certificate
+```c#
+// First connect to the DCC Gateway
```

Comment:
I just highlighted a few lines for commenting purposes, but I really like this whole section, nice :)